### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by spm2020spring**

### CODE REVIEW

**Comment by spm2020spring** [comment]
其他组员的信息也要添加。

**蓝珲**

File: readme.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ g1或g2如果出现空文件，对于本项目主题而言是没有意义的。
 
 # Contributor List
  
-(把每组所有成员的名字与学号写在下面)
+zjnuxrc-徐闰钞-201736900127
```
Comment (spm2020spring) [inline readme.md]: GitHub提示Merge时会发生冲突。

所以，你要现在你的本地机器上解决冲突，再Push上来。

**蓝珲**

Comment (spm2020spring) [reply readme.md]: @zjnuxrc 

GitHub提示Merge时会发生冲突。

所以，你要现在你的本地机器上解决冲突，再Push上来。

蓝珲

File: readme.md [hunk 2]
```diff
@@ -47,4 +47,6 @@ g1或g2如果出现空文件，对于本项目主题而言是没有意义的。
 
 # Contributor List
  
-zjnuxrc-徐闰钞-201736900127
+- zjnuxrc-徐闰钞-201736900127
+- yueyue0808-陈肖飞-201736900104
+- fangziai-方梓安-201736900105
```
Comment (spm2020spring) [inline readme.md]: @zjnuxrc 

`fangziai `不是一个GitHub账号。

蓝珲

**Comment by spm2020spring** [comment]
@zjnuxrc 

This branch has conflicts with the master branch.  Could you resolve them on your machine before you push?

Check the `git pull `command in this [tutorial ](https://github.com/spm2020spring/TeamCollaborationTutorial/blob/master/team.rst)

Hui

**Review by spm2020spring — COMMENTED** [review]
Looks much better now.

Hui

**Review by spm2020spring — COMMENTED** [review]
@yueyue0808

还是显示有Conflict.

蓝珲

**Comment by yueyue0808** [comment]
老师你上课修改了权限，我改了conflict后push不上去




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Software Project Management"<notifications@github.com&gt;;
发送时间:&nbsp;2020年4月3日(星期五) 中午1:02
收件人:&nbsp;"lanlab-org/GeneExpressionScatterPlot-WuTaiwei"<GeneExpressionScatterPlot-WuTaiwei@noreply.github.com&gt;;
抄送:&nbsp;"because of you"<1662727361@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [lanlab-org/GeneExpressionScatterPlot-WuTaiwei] Update readme.md (#2)





 
@spm2020spring commented on this pull request.
 
@yueyue0808
 
还是显示有Conflict.
 
蓝珲

**Comment by spm2020spring** [comment]
> 老师你上课修改了权限，我改了conflict后push不上去
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人:&nbsp;"Software Project Management"<notifications@github.com&gt;; 发送时间:&nbsp;2020年4月3日(星期五) 中午1:02 收件人:&nbsp;"lanlab-org/GeneExpressionScatterPlot-WuTaiwei"<GeneExpressionScatterPlot-WuTaiwei@noreply.github.com&gt;; 抄送:&nbsp;"because of you"<1662727361@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;; 主题:&nbsp;Re: [lanlab-org/GeneExpressionScatterPlot-WuTaiwei] Update readme.md (#2) @spm2020spring commented on this pull request. @yueyue0808 还是显示有Conflict. 蓝珲 — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.

@yueyue0808 
我改的是merge到远程master的权限，不是push的权限吧？你能把命令行的出错信息发出来看看吗？

蓝珲

**Comment by yueyue0808** [comment]
------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Software Project Management"<notifications@github.com&gt;;
发送时间:&nbsp;2020年4月3日(星期五) 中午1:19
收件人:&nbsp;"lanlab-org/GeneExpressionScatterPlot-WuTaiwei"<GeneExpressionScatterPlot-WuTaiwei@noreply.github.com&gt;;
抄送:&nbsp;"because of you"<1662727361@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [lanlab-org/GeneExpressionScatterPlot-WuTaiwei] Update readme.md (#2)





  
老师你上课修改了权限，我改了conflict后push不上去
 …
 ------------------&nbsp;原始邮件&nbsp;------------------ 发件人:&nbsp;"Software Project Management"<notifications@github.com&gt;; 发送时间:&nbsp;2020年4月3日(星期五) 中午1:02 收件人:&nbsp;"lanlab-org/GeneExpressionScatterPlot-WuTaiwei"<GeneExpressionScatterPlot-WuTaiwei@noreply.github.com&gt;; 抄送:&nbsp;"because of you"<1662727361@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;; 主题:&nbsp;Re: [lanlab-org/GeneExpressionScatterPlot-WuTaiwei] Update readme.md (#2) @spm2020spring commented on this pull request. @yueyue0808 还是显示有Conflict. 蓝珲 — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.
  
@yueyue0808
 我改的是merge到远程master的权限，不是push的权限吧？你能把命令行的出错信息发出来看看吗？
 
蓝珲

**Review by zjnuxrc — APPROVED** [review]

**Pull request merged**